contrib/pam_zfs_key/selinux/pam_zfs_key.if

```diff
@@ -0,0 +1,101 @@
+
+## <summary>policy for pam_zfs_key</summary>
+
+########################################
+## <summary>
+##	Allow domain to query datasets on ZFS.
+## </summary>
+## <param name="domain">
+##	<summary>
+##      Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`pam_zfs_key_query',`
+	gen_require(`
+		type device_t;
+	')
+
+	dnl Using ifdef() keeps `sepolgen-ifgen' from complaining
```

Comment:
Only systems with https://github.com/SELinuxProject/selinux/pull/348 applied will actually find this comment to be accurate.